osu.Server.Queues.ScoreStatisticsProcessor/Helpers/BatchInserter.cs

```diff
@@ -459,13 +499,6 @@ private async Task enqueueForFurtherProcessing(ulong firstId, ulong lastId, MySq
                 }
                 else
                 {
-                    if (dryRun)
-                    {
-                        // We can't retrieve this from the database because it hasn't been inserted.
-                        ScoreStatisticsItems.Add(new ScoreItem(new SoloScore(), new ProcessHistory()));
-                        return;
-                    }
```

Comment:
what was this ever doing :thinking: seems just straight wrong in retrospect

Reply:
It was working before `BatchInserter` was a thing. Just never conveyed across properly (only blocks queue insertion before this PR).